apps/explorer/src/components/transaction-card/TxCardUtils.tsx

```diff
@@ -70,7 +70,13 @@ export const genTableDataFromTxData = (
 ) => {
     return {
         data: results.map((txn) => ({
-            date: `${timeAgo(txn.timestamp_ms, undefined, true)} ago`,
+            date: (
+                <div
+                    style={{
+                        width: '85px',
```

Comment:
Why are we hard-coding this width?

Reply:
This reduces the jerkiness issue raised by Abhi.

Reply:
This reduces the jerkiness issue raised by Abhi.